test_runner/performance/pgvector/loaddata.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import numpy as np
+import pandas as pd
+import psycopg2
+from pgvector.psycopg2 import register_vector
+from psycopg2.extras import execute_values
+
+directory_path = (
+    "/Users/neon/work/vanilla/dbpedia-entities-openai3-text-embedding-3-large-1536-1M/data"
+)
+
+
+# Connection to PostgreSQL
+
+with psycopg2.connect(
+    "postgresql://neondb_owner:<secret>@ep-floral-thunder-w1gzhaxi.eu-west-1.aws.neon.build/neondb?sslmode=require"
+) as conn:
```

Comment:
I understand this script is an example, but let's make it reusable anyway. Let's make `directory_path` and `connstr` command line arguments, so we can upload data to any database.

Reply:
done